example/src/App.tsx

```diff
@@ -61,7 +61,15 @@ export default function App() {
 
     const asyncCreateLocalVideoTrack = async () => {
       try {
-        const createdLocalVideoTrack = await LocalVideoTrack.create();
+        const createdLocalVideoTrack = await LocalVideoTrack.create({
+          format: {
+            dimensions: {
+              width: 640,
+              height: 480,
+            },
+            framerate: 30,
```

Comment:
huge disappointment: despite this very flexible API, it seems like only a small set of dimension+framerate combinations are supported, and none of them are square.

womp womp.